backend/models/office_hours/oh_ticket.py

```diff
@@ -0,0 +1,38 @@
+from pydantic import BaseModel
+from datetime import datetime
+from ticket_type import TicketType
+from ticket_state import TicketState
+
+__authors__ = ["Sadie Amato, Bailey DeSouza, Meghan Sun, Maddy Andrews"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class OfficeHoursTicketDraft(BaseModel):
+    """
+    Pydantic model to represent an `OfficeHoursTicket` that has not been created yet.
+
+    This model is based on the `OfficeHoursTicketEntity` model, which defines the shape
+    of the `OfficeHoursTicket` database in the PostgreSQL database.
+    """
+
+    office_hours_event_id: int
```

Comment:
If we go down the road of slightly raising the level of abstraction, my vote here would be for this to be `office_hours_event: OfficeHoursEventPartial`

Reply:
This has been updated, thank you!